plugins/config-file.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: config-file               # plugin name must match your manifest file name (e.g. foo.yaml)
+spec:
+  version: "0.0.6"       # required, must be in semver format, prefixed with "v"
+  platforms:
+  # specify installation script for linux and darwin (macOS)
```

Comment:
do you mind cleaning up these comments? 
they are useful in the docs, but they don't serve a purpose here.

Reply:
Have cleaned up the comments. Should I change the version tag to "v0.0.6"

Reply:
Yeah, that would be good. I'll go ahead and update the developer guide.

Reply:
Looks like developer guide is already up-to-date with `v0.0.1`.

Reply:
I have updated this.